### PR TITLE
15_setup_structure

### LINKED ISSUE #15: Setup basic package structure
Building folder structure, setup file with metadata and license

### PR DESCRIPTION
**Pull request by EnricoGavagnin**
Closes #15 

This PR contains the basic structure for package development:

- folder structure
- setup.py 
- __init__.py
- LICENSE
- README.md

As guidelines I am following this two resources: [link1](https://packaging.python.org/en/latest/tutorials/packaging-projects/) and [link2](https://python-packaging.readthedocs.io/en/latest/minimal.html#).

### CODE REVIEW

**Review by emily-bicks — APPROVED** [review]
Looks good! Will definitely want to add a tests folder in future iterations, but this works as a starting point.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# nesta_ds_utils
+
+This repo contains a python package with basic data science utility functions to assist the DAP team.
```
Comment (georgerichardson) [inline README.md]:
```suggestion
A Python package with basic utility functions for data science projects at Nesta.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# nesta_ds_utils
+
+This repo contains a python package with basic data science utility functions to assist the DAP team. 
+
+### This package is currently under development
```
Comment (georgerichardson) [inline README.md:5]: Doesn't have to be this PR, but could be worth adding some contributor guidelines and some info on requesting/suggesting new features.

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[metadata]
+
+name = 'nesta_ds_utils'
+version ='0.1'
+description = 'Utility package for Nesta Data Science Practise',
```
Comment (georgerichardson) [inline setup.cfg]:
```suggestion
description = "Utility package for Nesta's Data Science Practice",
```

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+url = 'https://github.com/nestauk/nesta_ds_utils',
+author = 'Nesta DAP',
+author_email = enrico.gavagnin@nesta.org.uk,
```
Comment (georgerichardson) [inline setup.cfg]: 1. Is there a reason to use quote marks everywhere else but not here?
2. Is this best served by a personal email or the team email address?

**Review by georgerichardson — COMMENTED** [review]
Thanks for this first PR! Overall it looks good. I've made a few minor comments and a suggestion that could be addressed now or in future.

File: README.md (hunk 2 above)
Comment (EnricoGavagnin) [reply README.md:5]: Yes, that's a good point. I will open a new issue for that that can be addressed in a different PR, at least on the meantime we  have the basic structure there.

File: setup.cfg (hunk 4 above)
Comment (EnricoGavagnin) [reply setup.cfg]: Those were both very good points. Thank you for that.

**Pull request merged**